addon/mixins/scoped-query-cache.js

```diff
@@ -2,12 +2,15 @@ import Ember from 'ember';
 import sortedObject from 'ember-scoped-query-cache/utils/sorted-object';
 const { assert } = Ember;
 
+const CACHE_DECAY_TIMEOUT = 5000;
+
 export default Ember.Mixin.create({
```

Comment:
Routes primarily, but could be components too? I haven't really explored the use cases for components.